### DISCUSSION TITLE
Support setting the text color of the tooltip component

### DISCUSSION THREAD

**Discussion by badyan** [mantinedev/mantine#2940] [Feature requests]
It would be great if it were possible to change the color of the [tooltip](https://mantine.dev/core/tooltip/) text to some other colors.
The currently available props only allow controlling the background color.

Thank you for a wonderful library that we use extensively in our code!

**Comment by rtivital** [comment]
You can control tooltip color with `c` prop – https://codesandbox.io/s/suspicious-bas-r0ztbs?file=/src/App.tsx

**Comment by badyan** [comment]
Right, thanks!
Wasn't able to figure it out from the docs, maybe it's implied or something like that.
